app/views/account_reset/confirm_delete_account/show.html.erb

```diff
@@ -1,10 +1,10 @@
 <% title t('account_reset.confirm_delete_account.title') %>
 <div class="margin-y-2 padding-y-6 padding-x-4 tablet:padding-x-6 border border-error rounded-xl position-relative">
   <%= image_tag(
-        asset_url('status/error.svg'),
-        size: '48x48',
+        asset_url('status/delete.svg'),
+        size: '88x88',
         alt: t('errors.alt.error'),
```

Comment:
Is the error alt text still valid here, if the icon is being updated from "error" to "delete" ?